### PR TITLE
Ch all news jsonfile

### PR DESCRIPTION
**Pull request by amutava**
**What does this PR do?**
Combines all the json files into a single json file.
**How can this be tested?**
Run the command 
`python scraper.py`
**Issue**
[10](https://github.com/CodeForAfrica-SCRAPERS/taxclock_ke_news/issues/10)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,9 +1,52 @@
-# Ignore output of scraper
-data.sqlite
+# Byte-compiled / optimized / DLL files
 __pycache__/
 *.py[cod]
 *$py.class
+<<<<<<< HEAD
```
Comment (DavidLemayian) [inline .gitignore]: I'm guessing this isn't supposed to be here?

File: taxclock/__init__.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.
+                                   join(os.path.dirname(__file__), '..')))
+
```
Comment (DavidLemayian) [inline taxclock/__init__.py]: What are these lines of code for?

File: taxclock/scrapers/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import os 
+import sys
+
+sys.path.insert(0, os.path.abspath(os.path.
+                                   join(os.path.dirname(__file__), '..')))
```
Comment (DavidLemayian) [inline taxclock/scrapers/__init__.py]: And these. What are these lines of code for?

File: taxclock/scrapers/base.py [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
+import pytz
 
 from bs4 import BeautifulSoup
+from dateutil.parser import parse
+from datetime import datetime
 
 from taxclock import settings as env
 
-
+logging.basicConfig(filename=env.LOG_FILE, level=logging.INFO)
```
Comment (DavidLemayian) [inline taxclock/scrapers/base.py]: Why are we defining this here when we have it at `scraper.py`?

File: taxclock/scrapers/capital.py [hunk 5]
```diff
@@ -29,18 +30,26 @@ def scrape_page(self):
...
                         'img': img_url,
-                        'title': text
+                        'title': text,
+                        'date_published': date
                     })
-                self.aws_store(data, 'capital-news')
             except Exception as err:
-                log.error(str(err))
+                log.error(str(err) +
+                          ' :- ' + str(self.base.get_local_time()))
```
Comment (DavidLemayian) [inline taxclock/scrapers/capital.py]: Please remove local time item. It won't translate well to slack log handler.

File: taxclock/settings.py [hunk 6]
```diff
@@ -40,3 +40,8 @@
 # Placeholder image when none found on scrape
 IMG_PLACEHOLDER = 'https://github.com/CodeForAfrica/TaxClock/blob/kenya/img/\
                     placeholder.png'
+
+LOG_FILE = os.getenv('LOG_FILE', 'taxclock.log')
```
Comment (DavidLemayian) [inline taxclock/settings.py]: Let's not use a log file for logging please. I'm unsure why you've changed the logging approach significantly here.

File: taxclock/settings.py [hunk 7]
```diff
@@ -40,3 +40,8 @@
...
+
+# timeout for the url request.
+TIMEOUT = int(os.getenv('TIMEOUT', 30))
```
Comment (DavidLemayian) [inline taxclock/settings.py]: This env variable wouldn't be able to work on Moprh. It has to begin with `MORPH_`.

Also it doesn't follow the structure we've laid out above with the rest of the variables.

**Comment by amutava** [comment]
@DavidLemayian Kindly note that the issues raised on this pull request have been resolved.

**Pull request merged**